html-tables/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <th scope="col">Student Name</th>
+          <th scope="col">Course</th>
+          <th scope="col">Grade</th>
+        </tr>
+      </thead>
+      <tbody align="left">
+        <tr>
+          <th scope="row">Grace Hopper</th>
```

Comment:
these should all be tds in the body rows. The Th elements are just for the thead rows